dsgrid/config/annual_time_dimension_config.py

```diff
@@ -84,43 +85,38 @@ def build_time_dataframe(self) -> DataFrame:
         df_time = get_spark_session().createDataFrame(model_time, schema=schema)
         return df_time
 
-    def get_frequency(self) -> timedelta:
-        return timedelta(days=365)
+    def get_frequency(self) -> relativedelta:
+        freqs = [trange.frequency for trange in self.model.ranges]
+        assert set(freqs) == {freqs[0]}, freqs
```

Comment:
No asserts for possible user errors. 
```suggestion
        if len(set(freqs)) > 1:
            raise ...
```